lib/ui/platform_dispatcher.dart

```diff
@@ -168,6 +168,35 @@ class PlatformDispatcher {
   // A map of opaque platform view identifiers to view configurations.
   final Map<Object, ViewConfiguration> _viewConfigurations = <Object, ViewConfiguration>{};
 
+  /// The implicit [FlutterView] provided by the platform, if any.
```

Comment:
This feels a bit tautological.

I suspect that the "implicit view" is the view that, should the user not manually configure any other views, is the initial view automatically provided by Flutter itself, but @goderbauer and I were discussing this problem a few weeks back so I've got a bit of insider knowledge that an API doc reader may not have.

The name `implicitView` is probably fine, but we should define it clearly in the first line of the doc comment, which is all that's shown in some cases. 

Maybe something like:
```
The [FlutterView] in which content is rendered in the absence of any user-configured views.
```

Reply:
> I suspect that the "implicit view" is the view that, should the user not manually configure any other views, is the initial view automatically provided by Flutter itself

That's not quite right. It's a view provided for backwards compatibility reasons, or, by embedders that cannot create windows. See: https://github.com/flutter/engine/pull/39553#discussion_r1107565652